packages/marko-web-inquiry/i18n.js

```diff
@@ -0,0 +1,15 @@
+import translations from './translations';
+
+let value = '';
+function i18n(lang, fieldLabelKey) {
+  const keys = Object.keys(translations);
+  if (keys.includes(lang)) {
+    value = lang[fieldLabelKey];
+  } else {
+    throw new Error('LANGUAGE STRING MUST BE PROVIDED');
```

Comment:
Errors should give context to the developer of what caused the condition. In this case, provide the language and key that should have been defined. Something like this:
```
  if (!translations[lang]) {
    throw new Error(`No translations available for requested language ${lang}!`);
  }
  if (!translations[lang][key]) {
    throw new Error(`No translation available in ${lang} for requested key ${key}!`);
  }
  return translations[lang][key];
```